platform/web/package.json

```
@@ -5,18 +5,20 @@
   "description": "Development and linting setup for Godot's Web platform code",
   "scripts": {
     "docs": "jsdoc --template js/jsdoc2rst/ js/engine/engine.js js/engine/config.js js/engine/features.js --destination ''",
-    "lint": "npm run lint:engine && npm run lint:libs && npm run lint:modules && npm run lint:tools && npm run lint:html",
-    "lint:engine": "eslint \"js/engine/*.js\" --no-eslintrc -c .eslintrc.engine.js",
+    "lint": "npm run lint:engine && npm run lint:libs && npm run lint:modules && npm run lint:tools && lint:sw && npm run lint:html",
+    "lint:engine": "eslint \"../../misc/dist/html/*.js\" \"js/engine/*.js\" --no-eslintrc -c .eslintrc.engine.js",
```

```suggestion
    "lint:engine": "eslint \"../../misc/dist/html/*.js\" \"js/engine/*.js\" --no-eslintrc -c .eslintrc.engine.js",
    "lint:engine": "eslint \"js/engine/*.js\" --no-eslintrc -c .eslintrc.engine.js",
```

My bad, my patch incorrectly included this change